requirements.txt

```diff
@@ -1,3 +1,3 @@
-iribaker==0.1.2
-python-dateutil==2.8.1
-rdflib==5.0.0
+iribaker==git+https://github.com/CLARIAH/iribaker.git#egg=iribaker
```

Comment:
I raised an issue at https://github.com/CLARIAH/iribaker/issues about this ...

Reply:
See also https://github.com/opendata-swiss/ckanext-switzerland-ng/issues/209